src/assets/Comments.jsx

```diff
@@ -45,9 +45,8 @@ export default function Comments() {
                 }
                 return com;
             });
-            return newCommentData;
+            return sortByScore(newCommentData);

```

Comment:
No need to do it like this. You can just simply chain all array methods like this:
`const newCommentData = prevCommentData.map(...).filter().sort(...).map(...);`
You can chain them indefinitely and it would be the same as calling each method separately.